src/expression/comparison/__test__/unit/ge.test.ts

```diff
@@ -20,6 +20,11 @@ describe('Expression - Comparison - Greater Than or Equal', () => {
     // Truthy
     [operand(1), operand(0), true],
     [operand(1), operand(1), true],
+    // Truthy date cases
+    [operand('2023-01-01'), operand('2022-12-31'), true],
+    [operand('2023-01-01'), operand('2023-01-01'), true],
+    // Truthy date cases
```

Comment:
❓ Did you mean `False` here?

Reply:
Great catch, thank you @undees! The typo has been fixed in commit 2f84dfa5f66e77c9c4c423a27d3f00ba429e114d.